content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,251 @@
++++
+title = "Kubeflow on-prem in k8s multi-node cluster"
```

Comment:
> Thanks @alecglassford and @mastrogiovanni!
> 
> Answering your questions (quoted here for ease of reference):
> 
> > 1. I think this document might fit better under "Getting Started" rather than under "Use Cases." It could slot under "Kubeflow on Kubernetes"in the left-nav and have a title like "Kubeflow on Kubernetes (multi-node cluster)" — this doc basically tells readers to follow the "Kubeflow on Kubernetes" guide but also do some additional steps to make it work for a multi-node cluster. What do you think?
> 
> I agree - if this doc does follow directly on from the "Kubeflow on Kubernetes" guide then we should put it in the same section as that page. But I'm wondering: Has the [Kubeflow on Kubernetes](https://deploy-preview-897--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-k8s/) guide diverged since you wrote this PR, @mastrogiovanni? It no longer has the section about stuck pods.
> 
> > 1. Do you have a style recommendation for the term "on-prem"? The Google Developers style guide says `on-premises; not on prem or on premise` but there seem to be various usages on kubeflow.org. The terminology in this doc should be unified.
> 
> Thanks for pointing this one out! Personally, I don't like the recommendation in the Google Developers style guide that we should always use "on-premises" with the hyphen, no matter what the part of speech. It seems a clunky and bothersome solution that breaks English syntactical conventions. Still, I can see why the style guiders have chosen to retain the hyphen in all situations - it becomes very tricky to decide when to use the hyphen and when not. So, putting all that together, I think we should include the hyphen but use mostly the abbreviated form and explain the usage at the top of the page.
> 
> How about this:
> 
>     * Page title (as Alec suggested): `Kubeflow On-prem in a Multi-node Kubernetes Cluster`
> 
>     * First usage on the page:  `This guide describes how to set up Kubeflow on premises (on-prem) in ...`
> 
>     * All other occurrences on the page: `on-prem`